tests/test_filesystem.py

```diff
@@ -512,15 +506,15 @@ def test_user_wps_output_created_secure_data_proxy(self):
             "routes_to_create": ["wpsoutputs"],
             "permissions_cases": [(Permission.READ.value, Access.DENY.value, 0o660),
                                   (Permission.READ.value, Access.ALLOW.value, 0o664),
-                                  (Permission.WRITE.value, Access.ALLOW.value, 0o666),
+                                  (Permission.WRITE.value, Access.ALLOW.value, 0o664),
```

Comment:
Add some note next/above this new test case (and other below) indicating that `write` should be ignored by design, which is why `0o666` is not the expected value. Otherwise, it doesn't make sense in comparison with the other test cases why this value is expected.